bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -78,12 +83,53 @@ spec:
           name: ovn-cert
         - mountPath: /ovn-ca
           name: ovn-ca
+        - mountPath: /var/log/ovn
+          name: node-log
+        - mountPath: /dev/log
+          name: log-socket
+        - mountPath: /run/systemd/journal/dev-log
+          name: journald-socket
         terminationMessagePolicy: FallbackToLogsOnError
         resources:
           requests:
             cpu: 10m
             memory: 300Mi
 
+      - name: ovn-acl-logging
+        image: "{{.OvnImage}}"
+        command:
+        - /bin/bash
+        - -c
+        - |
+          set -euo pipefail
+
+          # Redirect err to null so no messages are shown upon rotation
+          tail -F /var/log/ovn/acl-audit-log.log 2> /dev/null & 
+          
+          #Rotate audit log files when then get to max size
+          MaxFileSize="{{.OVNPolicyAuditMaxFileSize}}"
+
+          while true
+          do
+
+          #Get size in bytes**
+          file_size=`du -b /var/log/ovn/acl-audit-log.log | tr -s '\t' ' ' | cut -d' ' -f1`
+          if [ $file_size -gt $MaxFileSize ];then
+            echo "Rotating OVN ACL Log File"
+            timestamp=`date '+%Y-%m-%dT%H-%M-%S'`
+            mv /var/log/ovn/acl-audit-log.log /var/log/ovn/acl-audit-log.$timestamp.log
+            ovs-appctl -t /var/run/ovn/ovn-controller.$(cat /var/run/ovn/ovn-controller.pid).ctl vlog/reopen
+          fi
+
+          done
```

Comment:
I'm not sure this is needed.  When I was looking at the OVN package last week, it looks like it already automatically configures logrotate to run `vlog/reopen` when rotation occurs.

Here's the upstream link to the logrotate config, but I saw it installed on my OpenShift cluster as well.

https://github.com/ovn-org/ovn/blob/master/rhel/etc_logrotate.d_ovn

or did you determine a reason why this may not work?

Reply:
Per our weekly meeting I was informed that the built in log rotate modules would not work out of the box with coreos images unfortunately.